python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -717,6 +717,33 @@ def test_write_batch_pickle_duplicate_symbols(arctic_library):
     assert not lib.list_symbols()
 
 
+def test_write_batch(arctic_library):
+    """Should be able to write different size of batch of data."""
+    lib = arctic_library
+    lib._nvs._lib_cfg.lib_desc.version.write_options.segment_row_size = 10
+    num_columns = 16
+    num_days = 200
+    num_symbols = 200
+    dt = datetime(2019, 4, 8, 0, 0, 0)
+    column_length = 6
+    num_rows_per_day = 1
+    list_requests = []
+    list_dataframes = {}
+    for sym in range(num_symbols):
+        columns = random_strings_of_length(num_columns, column_length, True)
+        df = generate_dataframe(random.sample(columns, 6), dt, num_days, num_rows_per_day)
+        list_requests.append(WritePayload("symbol_" + str(sym), df))
```

Comment:
As I said before, some of these should have metadata so that the test covers at least as much as the old one did (with a corresponding assert after the read)

Reply:
done